readthedocs/api/v3/serializers.py

```diff
@@ -375,6 +378,24 @@ def get_translations(self, obj):
         return self._absolute_url(path)
 
 
+class ProjectCreateSerializer(FlexFieldsModelSerializer):
+
+    """Serializer used to Import a Project."""
+
+    repository = RepositorySerializer(source='*')
+    homepage = serializers.URLField(source='project_url', required=False)
+
+    class Meta:
+        model = Project
+        fields = (
+            'name',
+            'language',
+            'programming_language',
+            'repository',
+            'homepage',
```

Comment:
We can add more fields here. Maybe these are also useful:

- default_branch
- default_version
- tags
- analytics_code
- show_version_warning